### ISSUE TITLE
Exact training set of FFDNet

### ISSUE THREAD

**Issue by shengyenlin** [cszn/FFDNet#33]
Hi,

Thanks for your amazing work.

I would like to confirm the exact training set of FFDNet, in your paper, you stated that "We collected a large dataset of source images, including 400 BSD images, 400 images selected from the validation set of ImageNet [53], and the 4,744 images from the Waterloo Exploration Database [54]".

I would like to ask where could I find the 400 BSD images and the 400 images selected from the validation set of ImageNet?

**Comment by Jeffery-MIC** [comment]
@shengyenlin Hi, I think you can find the 400 BSD dataset at [here](https://github.com/cszn/DnCNN/tree/master/TrainingCodes/DnCNN_TrainingCodes_v1.0/data/Train400). However, I couldn't find the 400 images selected from the validation set of ImageNet also. If you have found any helpful information, kindly request for your sharing. Thank you!
